README.md

```diff
@@ -2,43 +2,44 @@
 
 This module provides the required Azure resources to integrate an Azure subscription with Uptycs.
 
+## Prerequisites [Do Not Skip]
```

Comment:
i do not think **do no skip** is required.

Reply:
We have seen this common behaviour in a lot of our customers where they just brush past the Pre-requisites section without paying much attention to it. When we had a call with them, they said they did not notice this at all. Which is why I decided to move it to the top and add a cautionary message for them.